src/preloader/HttpServerElectron.ts

```diff
@@ -140,6 +140,9 @@ export class HttpServerElectron {
         rawTrailers: req.rawTrailers,
         method: req.method,
         url: req.url,
+        socket: {
+          localAddress: req.socket.localAddress,
```

Comment:
Are there other properties on `socket` we could pass through? Remote address, ports?

Reply:
https://nodejs.org/dist/latest-v16.x/docs/api/net.html#class-netsocket

I'm thinking the following:

`localAddress`
`localPort`
`remoteAddress`
`remotePort`

`localFamily` and and `remoteFamily` seem necessary but it's super easy to add these since that are all POJO values and not functions.

I'm gonna go with the 4 I named unless I hear otherwise. Feel free to propose others that seem useful.